config/wrapper/pci_input.conf

```diff
@@ -1,23 +1,92 @@
 [io_nvme_fvt]
+devices = interfaces:0
 disk = disks:0
+lv_disks = disks:0
+htx_disks = disks:1
 disks = disks:all
-device = interfaces:0
```

Comment:
we need this parameter for nvme functional test

Reply:
device = interface is still there Abdul point 2.

reason to reorder all the conf file was when we get the input file generated we have a standard way of data across devices and at the last we willhave only manually entered data for example passwordds